contrib/drivers/lvm/lvm.go

```diff
@@ -0,0 +1,290 @@
+// Copyright (c) 2017 OpenSDS Authors.
+//
+//    Licensed under the Apache License, Version 2.0 (the "License"); you may
+//    not use this file except in compliance with the License. You may obtain
+//    a copy of the License at
+//
+//         http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+//    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+//    License for the specific language governing permissions and limitations
+//    under the License.
+
+package lvm
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os/exec"
+	"strconv"
+	"strings"
+
+	log "github.com/golang/glog"
+	pb "github.com/opensds/opensds/pkg/dock/proto"
+	"github.com/opensds/opensds/pkg/model"
+	"github.com/opensds/opensds/pkg/utils/config"
+	"github.com/satori/go.uuid"
+	"gopkg.in/yaml.v2"
+)
+
+const (
+	vgName = "vg001"
+)
+
+var conf = LVMConfig{}
+
+type Driver struct {
+	config LVMConfig
+}
+
+type LVMConfig struct {
+	Pool map[string]PoolProperties `yaml:"pool,flow"`
+}
+
+type PoolProperties struct {
+	DiskType  string `yaml:"diskType"`
+	IOPS      int64  `yaml:"iops"`
+	BandWidth int64  `yaml:"bandwidth"`
```

Comment:
BandWidth or Bandwidth?

Reply:
It should be Bandwidth, but I didn't have so much energy to solve it because it's a bit complicated.